collectors/bzimport/tests/test_convertors.py

```diff
@@ -685,14 +685,14 @@ def get_flaw_bug(cls):
             "component": "vulnerability",
             "creation_time": "2000-01-01T12:12:12Z",
             "depends_on": [],
-            "description": "text",
```

Comment:
This data are supposed to be from Bugzilla, `description` there remains unchanged.